### PR TITLE
Hotfix/remove email error handling

### PR DESCRIPTION
**Pull request by chennan47**
<b>Purpose</b>
Improve the error handling method for remove emails.

### CODE REVIEW

**Comment by lbanner** [comment]
Nice work :+1: TY

File: website/profile/views.py [hunk 1]
```diff
@@ -112,11 +112,15 @@ def update_user(auth):
...
         for address in removed_emails:
-            if address == user.username:
-                raise HTTPError(httplib.FORBIDDEN)
             if address in user.emails:
-                user.remove_email(address)
-            user.remove_unconfirmed_email(address)
+                try:
+                    user.remove_email(address)
+                except PermissionsError as e:
+                    raise HTTPError(httplib.FORBIDDEN, e.message)
```
Comment (sloria) [inline website/profile/views.py]: In order to display the error message on the HTML error page you need to pass the message into the `data` argument

```python
raise HTTPError(.., data={'message_long': e.message})
```

Comment (sloria) [inline website/profile/views.py]: I will make this minor change during merge.

Comment (sloria) [inline website/profile/views.py]: Cancel that. Discussed with @chennan47 ; this message is not meant to be shown to the user, so it shouldn't be put into `data`.

**Pull request merged**